### ISSUE TITLE
Error Message: Unable to Find Template

### ISSUE THREAD

**Issue by purchasingtoolpak** [flyntwp/flynt#252]
Hi, I'm using Flynt Premium. I unzipped the file into the theme\flynt\Components\ folder.

I am getting the following error message:
Fatal error: Uncaught Twig\Error\LoaderError: Unable to find template 
"D:/Bitnami2/project-sunshine/apps/wordpress/htdocs/wp-content/themes/flynt/Components/PremiumComponents/HeroImageText/index.twig" 
(looked into: 
D:\Bitnami2\project-sunshine\apps\wordpress\htdocs\wp-content\themes\flynt, 
D:\Bitnami2\project-sunshine\apps\wordpress\htdocs\wp-content\themes\flynt\views, ,
D:\Bitnami2\project-sunshine\apps\wordpress\htdocs\wp-content\themes\flynt\Components\PremiumComponents/HeroImageText). 

in D:\Bitnami2\project-sunshine\apps\wordpress\htdocs\wp-content\themes\flynt\templates\page.twig:5 
	Stack trace: #0 D:\Bitnami2\project-sunshine\apps\wordpress\htdocs\wp-content\themes\flynt\vendor\twig\twig\src\Loader\FilesystemLoader.php(150): 
	Twig\Loader\FilesystemLoader->findTemplate('D:/Bitnami2/pro...') #1 D:\Bitnami2\project-sunshine\apps\wordpress\htdocs\wp-content\themes\flynt\vendor\twig\twig\src\Environment.php(299): 
	Twig\Loader\FilesystemLoader->getCacheKey('D:\\Bitnami2\\pro...') 
	#2 D:\Bitnami2\project-sunshine\apps\wordpre in D:\Bitnami2\project-sunshine\apps\wordpress\htdocs\wp-content\themes\flynt\templates\page.twig on line 5

The HeroImageText components file is in the directory. Although I did notice a back-slash right before the component name e.g. \PremiumComponent/HeroImageText . 
Any help debugging would be appreciated.

**Comment by domtra** [comment]
Hi @purchasingtoolpak ,
it is strange that this is only happening for PremiumComponents. I would expect the issue to come from something in the `functions.php` in the PremiumComponents folder.
As a quick fix, could you please try to move all of the premium components out of the PremiumComponents folder, one level higher, into the Components folder? Then copy the `getACFLayout` functions `PremiumComponents/functions.php` into the respective field group definition in `./inc/fieldGroups/`. As a last step, delete the PremiumComponents folder with the left over files.
I hope this solves your issue for now. This is option number 1 from the PremiumComponents readme. It is definitely the more robust method that we use most of the time. I will have a look at option 2 though, as well, to see how we can fix the issue you are facing.

**Comment by purchasingtoolpak** [comment]
OK, after following your instructions above the Premium Component is now working. However the images are not being generated in the Dynamic folder.  The img src path is.
`<img class="figure-image lazyautosizes lazyloaded" src="//localhost:3000/wordpress/wp-content/uploads/dynamic/2020/05/national-cancer-institute-eRExodEMiOE-unsplash-792x528-c-default.jpg" srcset="
            //localhost:3000/wordpress/wp-content/uploads/dynamic/2020/05/national-cancer-institute-eRExodEMiOE-unsplash-1584x1056-c-default.jpg 1584w,
            //localhost:3000/wordpress/wp-content/uploads/dynamic/2020/05/national-cancer-institute-eRExodEMiOE-unsplash-792x528-c-default.jpg 792w,
            //localhost:3000/wordpress/wp-content/uploads/dynamic/2020/05/national-cancer-institute-eRExodEMiOE-unsplash-375x250-c-default.jpg 375w" data-srcset="
            //localhost:3000/wordpress/wp-content/uploads/dynamic/2020/05/national-cancer-institute-eRExodEMiOE-unsplash-1584x1056-c-default.jpg 1584w,
            //localhost:3000/wordpress/wp-content/uploads/dynamic/2020/05/national-cancer-institute-eRExodEMiOE-unsplash-792x528-c-default.jpg 792w,
            //localhost:3000/wordpress/wp-content/uploads/dynamic/2020/05/national-cancer-institute-eRExodEMiOE-unsplash-375x250-c-default.jpg 375w" data-sizes="auto" alt="children-hospital" sizes="792px">`
However these images are not being generated in the Dynamic folder. I am working to resolve but any tips would be appreciated.

**Comment by purchasingtoolpak** [comment]
I would like to enable DynamicResize. I think this is a great feature of flynt. If you can provide further guidance to debug that would be appreciated.

**Comment by domtra** [comment]
@purchasingtoolpak there is a new work in progress branch called `dynamicResize` would you mind trying out if this works with your setup?
you will need to go into the `Global Options` and change the settings of `Timber Dynamic Resize`.

**Comment by purchasingtoolpak** [comment]
I installed the new branch using localwp on my desktop with no premium components. Then I installed the faker press plugin which added about 50 posts with images. I changed the dynamic resize global setting to yes to generate images on the fly.

No images rendered with the blog posts. I also created a Photo gallery using the slider component and no images rendered. 

I turned off dynamic image generation and the images did show for the blog posts and the photo gallery.

Any questions let me know.

Dave
